cmd/vcluster/cmd/start.go

```diff
@@ -184,7 +184,7 @@ func ExecuteStart(options *context2.VirtualClusterOptions) error {
 	}
 
 	// Ensure that service CIDR range is written into the expected location
-	err = wait.Poll(5*time.Second, 2*time.Minute, func() (bool, error) {
+	err = wait.PollImmediate(5*time.Second, 2*time.Minute, func() (bool, error) {
```

Comment:
Oh, there goes my - "Hey, I improved startup time by full 5 seconds" :joy: 
jk of course! I don't why I didn't notice that it was so slow to start.
Good catch!